src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__wrapper">
+        <div class="product-card__header">
+          <img
+            src="images/imac.jpeg"
+            alt="Apple laptop"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/5698f73d-66d7-485a-a351-4d0493d9ec13)

Reply:
ok